docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+
+services:
+  php:
+    build:
+      context: docker/php
+    working_dir: /var/www/packagist
+    volumes:
+      - .:/var/www/packagist
+    depends_on:
+      - mysql
+      - redis
+      - solr
+  mysql:
+    image: mysql:5.7
+    environment:
+      MYSQL_ROOT_PASSWORD: packagist
+  redis:
+    image: redis:3.2
+  solr:
+    build:
+      context: docker/solr
+    environment:
+      SOLR_CONFS: '/etc/solr' 
```

Comment:
This should be set in the Dockerfile IMO, as this path actually depends on the Dockerfile itself (and so overriding it does not make sense).
We should keep this knowledge in a single place.